src/sentry/tasks/post_process.py

```diff
@@ -171,7 +171,12 @@ def update_existing_attachments(event):
 
 @instrumented_task(name="sentry.tasks.post_process.post_process_group")
 def post_process_group(
-    is_new, is_regression, is_new_group_environment, cache_key, group_id=None, **kwargs
+    is_new,
+    is_regression,
+    is_new_group_environment,
+    super_important_non_generic_ass_event_processing_store_cache_key,
```

Comment:
We can't change the task arg name. :)

Reply:
Rats.

Reply:
is `_dispatch_post_process_group_task` the only place this is called? I could change the name there.

Reply:
Also got some strong opinions on the fact that this is not actually a "cache key".